cmake/FindHerbert.cmake

```diff
@@ -7,9 +7,12 @@ Looks for the Herbert Matlab software package.
 Variables defined by the module
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-``Herbert_ROOT``
+``Herbert_CORE``
```

Comment:
They are independent. It will attempt to detect the `herbert_init.m` in `Herbert_CORE` and if that fails, it will error out. If it succeeds, it will add `Herbert_CORE` and `Herbert_ROOT` into the cache, `Herbert_ROOT` as the one defined by the user.